readme.md

```diff
@@ -1,44 +1,95 @@
 ## Repository Purpose:
-This repository contains code used to generate synthetic LAR files. These files will be used to support the development of the 2018 HMDA data submission platform. The success of the HMDA program requires that the data submitted meet the technical and business requirements of the HMDA Platform. The code and test files in this repository will provide a standardized means of checking the implementation of the business rules for data submission in the HMDA Platform.
+This repository contains code used to generate synthetic LAR files. These files will be used to support the development of the 2018 HMDA data submission platform and platforms in the future. The success of the HMDA program requires that the data submitted meet the technical and business requirements of the HMDA Platform. The code and test files in this repository will provide a standardized means of checking the implementation of the business rules for data submission in the HMDA Platform.
 
 ## Repository Use Cases:
 - Enable business analysts to test the 2018 HMDA Platform
 - Provide files to check API returns
 - Provide a resource for developers to use while implementing business rules in code
 - Provide synthetic data files meeting the 2018 HMDA schema to be used in training
 
+## Repository Structure
+- [Python](https://github.com/cfpb/hmda-test-files/tree/master/python):
+    - Contains the notebooks and python scripts used to generate synthetic LAR data, perturb the data so that it fails specific business rules, and code to check that the produced files fail as expected.
```

Comment:
Can you add "edits" after `produced files fail` ?

Reply:
Sure! This change will be reflected in the next commit.